### ISSUE TITLE
Resolve insertion in QA255.016-VL

### ISSUE THREAD

**Issue by eharkins** [matsengrp/cft#285]
investigating 3bp insertion in cdr3 of QA255.016-VL 

This is becoming more standard after #279 #277 #281 etc. #281 means we now have an automated method to detect such indels. We will do a similar analysis to 157-Vk where we see where the indel sequences cluster among the larger tree. I will create a separate issue for generalizing the script I use to do that.

**Comment by eharkins** [comment]
@lauradoepker said: 
> super clear results re: the indel clade in the 016-VL clonal family:
![image](https://user-images.githubusercontent.com/12140437/60989694-77170880-a2fb-11e9-85e6-8b533bdb8670.png)
![image](https://user-images.githubusercontent.com/12140437/60989726-86965180-a2fb-11e9-9d79-bb5cad8f96d7.png)
> @matsen, based on this, I think the next step is that we’ll need to figure out which ancestral sequence lies at the internal node where the insertion clade branches off.  This will inform “when” the insertion happened in context of the ecgtheow lineage that I’m working with.

@matsen said:
> @lauranoges this is quite interesting. It seems to me that we need to figure out the sequence of mutations in the insertion if you are going to reconstruct it, right?
> @psathyrella, want to tell Laura about the results of your no-indel analysis?

@psathyrella said: 
> oh, yeah, so there is no other v gene that aligns to the sequences if we forbid indels. It just makes the v_3p deletion longer

I believe this is where we left off. Just noting this so we can update / close this issue when we meet momentarily

**Comment by lauradoepker** [comment]
Summary of this issue:
We discovered that the seed that was submitted to partis has an insertion within the CDR3.  This was missed because I was using the seed that was output from ecgtheow, which uses indel-reversed sequences.  Once the mistake was caught, @eharkins ran an indel-filtering script/protocol that identified all sequences that contained an insertion at the right spot (the same spot as the seed) from 016-VL's best partition seeded cluster.  These sequences were highlighted in red within the whole clonal family using FastTree.  @eharkins also used indel-reversed sequences to seed prune the clonal family to 100 sequences and, again, highlighted any sequences that actually contain the insertion (in red) - not sure if this is a FastTree or ML tree.

Takeaways for this indel event:  The larger clonal family FastTree shows that most red sequences cluster, though there are some spurious sequences that don't cluster with the rest.  These spurious sequences were identified and they DO have the right insertion, but are a little weird (highly mutated or PCR error).  Ultimately not sure why they don't cluster with the rest.  The main group of red sequences all contain the "correct" indel and it appears that most of these sequences have inserted a Serine residue, while the mature seed sequence underwent a subsequent mutation event and changed this inserted Serine to an Asparagine.

Based on the most recent common ancestor node preceding the first red sequence, the insertion event happened before most of our major ecgtheow-determined mutation events, and thus is relevant for us to add to our "lineage intermediate" antibodies for biological tests.  For our intermediates that are most biologically interesting, there is an NGS sequence that is almost identical (yay validation!) and its insertion is a Serine (not an Asparagine).  Thus, for our intermediates, I'll be adding in a Serine insertion even though the mature sequence has an Asparagine.  Based on the fact that no other NGS has mutated the insertion codon to an Asparagine, I deduce that this mutation event happened late in the lineage and thus it will be lumped in with the other late mutations that occurred right at the end of the ecgtheow lineage to achieve the final seed 016-VL sequence.
